root/layout/components/Footer.js

```diff
@@ -9,87 +9,568 @@
-        <a className="internal" href="https://blog.metabrainz.org/">{l('Blog')}</a>
-        <a className="internal" href="https://twitter.com/MusicBrainz">{l('Twitter')}</a>
-
-        {DBDefs.BETA_REDIRECT_HOSTNAME ? (
-          <a
-            className="internal"
-            href={
-              '/set-beta-preference?' + returnToCurrentPage($c)
-            }
-          >
-            {DBDefs.IS_BETA
-              ? l('Stop using beta site')
-              : l('Use beta site')}
-          </a>
-        ) : null}
-      </p>
-
-      <p className="right">
-        {exp.l(
-          `Brought to you by {MeB|MetaBrainz Foundation} and our
-           {spon|sponsors} and {supp|supporters}. Cover Art provided
-           by the {caa|Cover Art Archive}.`,
-          {
-            caa: '//coverartarchive.org/',
-            MeB: 'https://metabrainz.org/',
-            spon: 'https://metabrainz.org/sponsors',
-            supp: 'https://metabrainz.org/supporters',
-          },
-        )}
-
-        {DBDefs.DB_STAGING_SERVER && DBDefs.GIT_BRANCH ? (
-          <>
+    <section className="footer">
+      <div className="ms-4 me-4 mb-3 ps-4 pb-3 pe-4 border-top container">
+        <div className="row mb-4 mt-2 pt-2">
+          <div className="col-sm-12 col-md-4">
+            <h3>
+              <img
+                alt="MusicBrainz"
+                src="../../static/images/meb-logos/musicbrainz.svg"
+                width="180"
+              />
+            </h3>
             <br />
-            {exp.l('Running: {git_details}', {
-              git_details: (
-                <span
-                  className="tooltip"
-                  key="git_details"
-                  title={DBDefs.GIT_MSG}
-                >
-                  {DBDefs.GIT_BRANCH}
-                  {' '}
-                  {bracketedText(DBDefs.GIT_SHA)}
+            <p className="fs-4">
+              {l(`MusicBrainz is an open music encyclopedia that collects
+              music metadata and makes it available to the public.`)}
+            </p>
+            <ul className="list-unstyled">
+              <li>
+                <span className="fs-4">
+                  {l('Development IRC: ')}
+                </span>
+                <a
+                  className="fw-bold fs-4"
+                  href="https://kiwiirc.com/nextclient/irc.libera.chat/?#metabrainz"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('#metabrainz')}
+                </a>
+              </li>
+              <li>
+                <span className="fs-4">
+                  {l('Email: ')}
                 </span>
-              ),
-            })}
-          </>
-        ) : null}
+                <a
+                  className="fw-bold fs-4"
+                  href="mailto:support@metabrainz.org"
+                >
+                  {l('support@metabrainz.org')}
+                </a>
```

Comment:
Similarly, we probably just want a contact link to `https://metabrainz.org/contact` so that people read that before contacting us? @reosarevok would have more relevant input than me since he deals with support.